haystack/components/converters/multi_file_converter.py

```diff
@@ -114,3 +117,20 @@ def __init__(self, encoding: str = "utf-8", json_content_key: str = "content") -
         self.pipeline = pp
         self.output_mapping = {"joiner.documents": "documents", "router.unclassified": "unclassified"}
         self.input_mapping = {"sources": ["router.sources"], "meta": ["router.meta"]}
+
+    if TYPE_CHECKING:
+        # fake method, never executed, but static analyzers will not complain about missing method
+        def run(  # noqa: D102
+            self,
+            *sources: list[Union[str, Path, ByteStream]],
```

Comment:
```suggestion
            *,
            sources: list[Union[str, Path, ByteStream]],
```

this should be the right signature

Reply:
typo